baseapp-core/testproject/requirements.txt

```diff
@@ -1,7 +1,7 @@
 # install "install_requires" from setup.cfg
 -e ./baseapp-core
 
-django==3.2
+django==3.2.20
```

Comment:
I think we can remove django from here now since its comming from the github actions versions matrix
```suggestion
```